python/README.md

```diff
@@ -65,7 +65,10 @@ More extensive documentation (generated with Sphinx) is available in the `python
 ## Changelog
```

Comment:
This is not directly related to this PR. 
But I wondering given that scikit-learn (and other python libraries) use `joblib.Parallel` heavily, was implementing a `joblib.Parallel` backend considered in the original design? If so, what's the reason that it was not taken? Thanks!